### PR TITLE
Show finalizing view for sd sync

### PR DESCRIPTION
**Pull request by anna1901**
https://trello.com/c/JxCEN4rV

### CODE REVIEW

File: AirCasting/SDSync/ViewsFlow/SDSyncingAB/SyncingABView.swift [hunk 1]
```diff
@@ -61,7 +65,14 @@ extension SyncingABView {
     }
     
     var titleLabel: some View {
-        Text(progressLabel ?? "")
+        VStack(alignment: .leading) {
+            if viewModel.isDownloadingFinished {
+                Text("Finalizing...")
+            } else {
+                progressTitle != nil ? Text("Syncing " + progressTitle!.lowercased()) : Text("Syncing...")
+                Text(progressCount ?? "")
```
Comment (jankrzempek) [inline AirCasting/SDSync/ViewsFlow/SDSyncingAB/SyncingABView.swift:73]: Strings please 😄

**Review by jankrzempek — COMMENTED** [review]
Only those Strings please

File: AirCasting/ABConnector/SDSyncController.swift [hunk 2]
```diff
@@ -32,7 +32,7 @@ class SDSyncController: ObservableObject {
         self.sessionSynchronizer = sessionSynchronizer
     }
     
-    func syncFromAirbeam(_ airbeamConnection: CBPeripheral, progress: @escaping (SDCardSyncProgess) -> Void, completion: @escaping (Bool) -> Void) {
+    func syncFromAirbeam(_ airbeamConnection: CBPeripheral, progress: @escaping (SDCardSyncProgess) -> Void, finalizing: @escaping () -> Void, completion: @escaping (Bool) -> Void) {
```
Comment (pawelgil) [inline AirCasting/ABConnector/SDSyncController.swift:35]: I think maybe switching from SDCardSyncProgress to something like:
```Swift
enum SDCardSyncStatus {
    case inProgress(SDCardSyncProgress)
    case finalizing
}
```
and having just a `progress` and `completion` blocks will be better? It looks a bit odd, I didn't quite get what is a difference between finalizing and completion at first

**Review by pawelgil — APPROVED** [review]
I left a comment with suggestion, but leaving it as approved because it's optional

**Pull request merged**